shell/platform/darwin/macos/framework/Source/FLEEngine.mm

```diff
@@ -123,7 +123,7 @@ static void OnPlatformMessage(const FlutterPlatformMessage* message, FLEEngine*
 
 @implementation FLEEngine {
   // The embedding-API-level engine object.
-  FlutterEngine _engine;
+  EmbedderFlutterEngine _engine;
```

Comment:
Don't you want to use macro here too?

Reply:
I didn't see a need to support compiling this with alternate prefix versions, so I just used the modified name directly. If you think it would be more readable with the macro (e.g., for searching) I can change it.

Reply:
Yeah, I was thinking it would be easier to read. But this is fine too.

Reply:
Done, I think the readability argument is a good one.